src/main/java/com/afornalik/ox/board/BoardField.java

```diff
@@ -0,0 +1,18 @@
+package com.afornalik.ox.board;
+
+class BoardField {
+
+    private FieldStatus fieldStatus;
+
+    BoardField() {
+        this.fieldStatus = FieldStatus.EMPTY;
+    }
+
+    FieldStatus showStatus() {
+        return fieldStatus;
+    }
+
+    void changeStatus(FieldStatus newFieldStatus) {
+        this.fieldStatus = newFieldStatus;
+    }
```

Comment:
not needed if use non-static map, if you use this function only for change EMPTY to X or O